### ISSUE TITLE
Error compiling scss

### ISSUE THREAD

**Issue by Geczy** [DesignRevision/shards-ui#12]
```js
@import '~bootstrap/scss/bootstrap';
@import '~shards-ui/src/scss/shards';
```

I receive the following error located here

https://github.com/DesignRevision/shards-ui/blob/master/src/scss/_variables.scss#L744

```
$popover-arrow-outer-width:           ($popover-arrow-width + 1px) !default;
                                     ^
      Incompatible units: 'px' and 'rem'.
```

**Comment by hisk** [comment]
The `$popover-arrow-width` variable uses `px` [a few lines above](https://github.com/DesignRevision/shards-ui/blob/master/src/scss/_variables.scss#L740) in Shards.

Please make sure that you are using the `beta-1` version of Bootstrap 4 or that you are not changing the value type somewhere in between.

**Comment by hisk** [comment]
Oh, haven't noticed that, thanks for pointing it out. Please let me know if using `beta-1` fixes your issue.

**Comment by Geczy** [comment]
It does fix it. Rolling back is not a fix though, do you plan to support bootstrap 4b2?

**Comment by hisk** [comment]
Adding support for beta-2 is on the list, but not a priority yet. If v4-stable doesn't ship by then, expect some movement in that direction starting early next year.

**Comment by dlucian** [comment]
At least you might consider changing the README.md, otherwise anyone who tries to install it via npm will hit this error. I bumped into it a few moments ago, good thing I was subscribed to the issues and was aware of this.

**Comment by piuccio** [comment]
I'm having this same problem when loading bootstrap `4.3.1`

**Comment by tajchumber** [comment]
me too, can we reopen this issue please?

**Comment by hisk** [comment]
@piuccio @tajchumber Can you please share the error you are receiving here?

**Comment by tajchumber** [comment]
@hisk The issue is [here](https://github.com/DesignRevision/shards-ui/blob/1433749e614e8dddc836c6c09cc818141fc78d7f/src/scss/_variables.scss#L759)

The error I get is:
```ERROR in ./node_modules/mini-css-extract-plugin/dist/loader.js!./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js!./scss/admin-section/app.scss
Module build failed: ModuleBuildError: Module build failed:
$popover-arrow-outer-width:           $popover-arrow-width + 1px !default;
                                     ^
      Incompatible units: 'px' and 'rem'.
```

**Comment by piuccio** [comment]
I encountered this problem while trying to use `shard-dashboard-lite` inside a next.js project using sass.

If I import `@import "shard-dashboard-lite/src/shards-dashboard/styles/shards-dashboards.1.1.0.min.css";` everything works fine, but when I use `@import "shard-dashboard-lite/src/shards-dashboard/styles/scss/shards-dashboards.scss";` I get the error

```
$popover-arrow-outer-width:           $popover-arrow-width + 1px !default;
                                     ^
      Incompatible units: 'px' and 'rem'.
      in .../node_modules/shards-ui/src/scss/_variables.scss (line 759, column 39)
```

I'm using
```
"bootstrap": "4.3.1",
"node-sass": "4.11.0",
"shards-react": "1.0.3"
```

**Comment by hisk** [comment]
@tajchumber @piuccio The issue is cause due to the fact that Shards does not currently support Bootstrap 4.3.x. Please check and make sure that you are using Bootstrap 4.1.x 😄

Support for the latest version is coming soon.

**Comment by tajchumber** [comment]
@hisk this problem persists with Bootstrap 4.1.3 as well

**Comment by Dronax** [comment]
@hisk I have a same problem. I use bootstrap: 4.3.1. Please fix it

**Comment by aalimovs** [comment]
Same errors here:

**4.3.1** - Error: Incompatible units: 'px' and 'rem'.
**4.1.3** - Error: Incompatible units: 'px' and 'rem'.
**4.0.0-beta** - Error: (en: "Choose file...") isn't a valid CSS value

**Comment by ElmirMahmudov** [comment]
i just imported bootstrap after 'shards-dashboards' it worked.

**Comment by cookieshell** [comment]
There are known factors that can cause this issue depending on the environment and dependencies versions. Our team will provide a fix as soon as possible. Thank you for your patience!

**Comment by iBelit** [comment]
problem is with this line:759
$popover-arrow-outer-width:           $popover-arrow-width + 1px !default;

i removed "!default" from $popover-arrow-width variable in line 755 and my problem solved!

**Comment by cookieshell** [comment]
Fixed in #57

**Comment by MD-AZMAL** [comment]
Having the same issue

code 
```
@import '~bootstrap/scss/bootstrap';
@import '~shards-ui/src/scss/shards';
```

Version
```
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "shards-react": "^1.0.3",
    "bootstrap": "^4.6.0",
    "node-sass": "^5.0.0",
```
